aexpect/client.py

```diff
@@ -1311,11 +1311,16 @@ def cmd_status_output(self, cmd, timeout=60, internal_timeout=None,
             # Send the 'echo $?' (or equivalent) command to get the exit status
             status = self.cmd_output(self.status_test_command, 30,
                                      internal_timeout, print_func, safe)
+            # Escape sequence for Bracketed Paste Mode
+            escape_sequence = r'\x1b\[\?[0-9;]*[hl]'
+
+            # Remove the Bracketed Paste Mode escape sequence using re.sub
+            clean_status = re.sub(escape_sequence, '', status)
```

Comment:
I think it'd be more consistent to rely on `aexpect.utils.astring.strip_console_codes`. But apart from that it makes sense to strip extra characters that might be present...

Reply:
Thank you for the review @ldoktor! I will use this function and try once.